zerver/tests/test_home.py

```diff
@@ -465,7 +466,7 @@ def test_num_queries_with_streams(self) -> None:
         self._get_home_page()
 
         # Then for the second page load, measure the number of queries.
-        with self.assert_database_query_count(44):
+        with self.assert_database_query_count(45):
             result = self._get_home_page()
```

Comment:
These database queries are starting to add up a bit; we should look at whether there's a refactor we can/should do.

Not a blocker for this PR; I'll open an issue.

Reply:
#31028